test/autoschedulers/mullapudi2016/param.cpp

```diff
@@ -119,21 +93,42 @@ int main(int argc, char **argv) {
         return 0;
     }
 
-    if (argc != 2) {
-        fprintf(stderr, "Usage: %s <autoscheduler-lib>\n", argv[0]);
+    if (argc < 2) {
+        fprintf(stderr, "Usage: %s <autoscheduler-lib> [--gpu]\n", argv[0]);
         return 1;
     }
 
     load_plugin(argv[1]);
 
+    // We don't actually care about the content of the 3rd argument. Enable
+    // experimental GPU scheduling whenever a 3rd argument is present.
+    const bool using_experimental_gpu_schedule = (argc >= 3);
+    const Halide::AutoschedulerParams autoscheduler_params{"Mullapudi2016",
+                                                           {{"experimental_gpu_schedule", using_experimental_gpu_schedule ? "1" : "0"}}};
+
+    Target target = get_jit_target_from_environment();
+    if (using_experimental_gpu_schedule) {
+        target.set_feature(Target::CUDA);
```

Comment:
Why not `if (target.has_gpu_feature()) { experimental_gpu_schedule = true; }`?

Reply:
Buildbots will run all tests without GPU, and then again with CUDA GPU, and then again with OpenCL GPU, and then again with Vulkan GPU. Similar things happen on the macos build bots which run the test suite with CPU and then again with GPU Metal.

Your tests define whether experimental_gpu_schedule is 1 or 0. Not the build bots.